### PR TITLE
alex: add page

### PR DESCRIPTION
**Pull request by Aracki**
I have added [alex](https://github.com/get-alex/alex) - a tool for catching insensitive, inconsiderate writing.

### CODE REVIEW

File: pages/common/alex.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# alex
+
+> Tool that catch insensitive, inconsiderate writing.
```
Comment (pxgamer) [inline pages/common/alex.md]:
```suggestion
> A tool that catches insensitive, inconsiderate writing.
```

File: pages/common/alex.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+> It helps you find gender favouring, polarising, race related, religion inconsiderate, or other unequal phrasing in text.
+> Homepage: <https://github.com/get-alex/alex>.
+
+- Analyze text:
```
Comment (pxgamer) [inline pages/common/alex.md]:
```suggestion
- Analyze text from `stdin`:
```

File: pages/common/alex.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`echo {{His network looks good}} | alex --stdin`
+
+- Analyze all files in the current directory:
+
+`alex`
+
+- Analyze file:
```
Comment (pxgamer) [inline pages/common/alex.md]:
```suggestion
- Analyze a specific file:
```

**Comment by schneiderl** [comment]
I did not knew about alex. This looks awesome. 👍 

Thank you for the new page by the way.

**Comment by Aracki** [comment]
@schneiderl I often find interesting tools when going through `tldr` repo.

**Pull request merged**